2019_02_meetup.md

```diff
@@ -2,15 +2,11 @@
 
 - Who: folks interested in building the Linux kernel w/ Clang.
 - What: a weekend meetup
+  - [Schedule](https://sites.google.com/google.com/clang-built-linux-meetup/home)
```

Comment:
This link is 404, is it not live yet?

Reply:
can you please check again? @gwelymernans just updated the permissions.